src/go/k8s/pkg/resources/featuregates/shadow_indexing.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+// Package featuregates gives ability to control k8s resource creation
+// based on the Redpanda version
+package featuregates
+
+import "github.com/Masterminds/semver/v3"
+
+const (
+	major = uint64(21)
+	minor = uint64(10)
+)
+
+// ShadowIndex feature gate should be removed in 3 version starting
+// from v21.10.x where cloud cache directory was introduced
```

Comment:
will there be 10 version since it's already november? xD

Reply:
It doesn't matter because any version that is 10 or greater.